### DISCUSSION TITLE
Yahoo auth provider

### DISCUSSION THREAD

**Discussion by biblebreeze** [supabase/supabase#22639] [Feature Requests]
Please add Yahoo in the Providers for authorization. I have a small group of users whom uses them. Yahoo can be accessed through oauth 2.0 and openid.

https://developer.yahoo.com/oauth2/guide/

**Comment by TheOtherBrian1** [comment]
It may be better to cross-post this in the [External OAuth Provider Requests discussion
](https://github.com/supabase/auth/issues/451)
